features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Configuration/Topics.jsx

```diff
@@ -290,28 +288,38 @@ export default function Topics(props) {
     }
     const onMarkAsDelete = useCallback(onOperationSelectM, [setSelectedOperation]);
 
+    /**
+     *
+     * @param {*} spec
+     */
+    function verifySecurityScheme(spec) {
+        /* eslint-disable no-param-reassign */
```

Comment:
Please use an alternative variable other than param, and try not to disable eslint rules